packages/web/src/components/buy-sell-modal/components/TokenDropdown.tsx

```diff
@@ -93,153 +102,167 @@ const CustomOption = (props: OptionProps<TokenOption>) => {
+        css={{
+          height: spacing.unit16,
+          cursor: disabled ? 'not-allowed' : 'pointer',
+          opacity: disabled ? 0.5 : 1,
+          '&:hover': !disabled
+            ? {
+                backgroundColor: color.background.surface2
+              }
+            : undefined
         }}
-        styles={{
-          container: (provided) => ({
-            ...provided,
-            width: '100%',
-            height: '100%'
-          }),
-          control: (provided) => ({
-            ...provided,
-            backgroundColor: 'transparent',
-            border: 'none',
-            boxShadow: 'none',
-            minHeight: '100%',
-            height: '100%',
-            display: 'flex',
-            alignItems: 'center',
-            cursor: disabled ? 'not-allowed' : 'pointer',
-            '&:hover': {
-              border: 'none'
-            },
-            paddingLeft: spacing.m,
-            paddingRight: spacing.m
-          }),
-          menuPortal: (provided) => ({
-            ...provided,
-            zIndex: zIndex.TOAST
-          }),
-          menu: (provided) => ({
-            ...provided,
-            backgroundColor: color.background.white,
-            boxShadow: shadows.far,
-            borderRadius: cornerRadius.m,
-            padding: `${spacing.s} 0`,
-            border: `1px solid ${color.border.default}`,
-            minWidth: 300,
-            right: 0,
-            marginTop: spacing.l
-          }),
-          menuList: (provided) => ({
-            ...provided,
-            padding: spacing.s,
-            maxHeight: 200,
-            overflowY: 'auto'
-          }),
-          option: (provided) => ({
-            ...provided,
-            backgroundColor: 'transparent',
-            color: color.text.default,
-            padding: 0,
-            '&:hover': {
-              backgroundColor: color.background.surface2,
-              cursor: 'pointer'
-            },
-            '&:active': {
-              backgroundColor: color.background.surface2
-            }
-          }),
-          singleValue: (provided) => ({
-            ...provided,
-            margin: 0,
-            padding: 0
-          }),
-          valueContainer: (provided) => ({
-            ...provided,
-            margin: 0,
-            padding: 0
-          })
+      >
+        <Flex
+          gap='s'
+          alignItems='center'
+          css={{ paddingLeft: spacing.m, paddingRight: spacing.m }}
+        >
+          <TokenIcon
+            logoURI={selectedToken.logoURI}
+            icon={selectedToken.icon}
+            size='2xl'
+            hex
+          />
+          <IconCaretDown size='s' color='default' />
+        </Flex>
+      </Flex>
+      <Menu
+        isVisible={isOpen}
+        anchorRef={wrapperRef}
+        css={{
+          border: 'none',
+          boxShadow: 'none',
+          backgroundColor: 'transparent',
+          maxWidth: 280
```

Comment:
think this should just be the width, not max

Reply:
yea i wanted to do that too, but there was some weirdness there so this felt better in the mean time.